### PR TITLE
I finish the form

### PR DESCRIPTION
**Pull request by Dutervil**
Please @lnball , have a look at this

### CODE REVIEW

File: configuration/pih/scripts/global/mch.js [hunk 1]
```diff
@@ -27,11 +27,11 @@ function setUpEdd(currentEncounterDate, msgWeeks) {
       let yearMonthDay = lastPeriodDateValue.split('-');
       if (yearMonthDay.length == 3) {
         lastPeriodDate.setFullYear(yearMonthDay[0]);
-        lastPeriodDate.setMonth(+yearMonthDay[1] -1); // the month starts from 0 for January
+        lastPeriodDate.setMonth(+yearMonthDay[1] - 1); // the month starts from 0 for January
```
Comment (lnball) [inline configuration/pih/scripts/global/mch.js:30]: Does this break this?  Do you need this space?

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:1]: I suggest creating this file in the zl repo.  Do not change the one in openmrs-config-pihemr.  Same issue with the vitals.  It's possible that the vtials form is used by other countries.  (I think that Sierra Leone and Liberia have their own mental health form).

**Review by lnball — CHANGES_REQUESTED** [review]
@Dutervil See my comments.

File: configuration/pih/scripts/global/mch.js (hunk 1 above)
Comment (Dutervil) [reply configuration/pih/scripts/global/mch.js:30]: I did not touch this code

Comment (lnball) [reply configuration/pih/scripts/global/mch.js:30]: It's showing the line changed by you.  Just remove the extra space.

image

Comment (Dutervil) [reply configuration/pih/scripts/global/mch.js:30]: Sure . I will

**Pull request merged**